README.md

```diff
@@ -82,34 +82,37 @@ Week 02 _(~1.5 hours)_
 
 - Linting
 - Continuous integration
+
+Week 03 _(~2.5 hours)_
```

Comment:
All good. We should ping Albina that it has been extended by a week because the first weeks were a bit too heavy.

Reply:
Just a clarification here: at this point, I'm evolving the Daredata branch separately from the NOS branch as it was dawning on me that their constraints are different. The NOS repo, in fact, has 10 weeks and not 9 (see: https://github.com/nosportugal/faast-foundations/pull/1). I got pushback whenever a unit went beyond 3 weekly hours over there. On the other hand, our pods are super motivated and will do the courses on nights and weekends if they need to.

Another change exclusive to the NOS repo is that I switched the Type Hint content for something more basic and easier to digest.

PS: I already gave Albina a heads up that the week suggestion might change depending on the feedback of the first pods 😉